CHANGELOG.md

```diff
@@ -18,6 +18,19 @@ how a consumer would use the library (e.g. adding unit tests, updating documenta
 
 - Python 3.9 support.
 
+- `sdk.cases` methods:
+    - `sdk.cases.create()`
+    - `sdk.cases.get_all()`
+    - `sdk.cases.export_summary()`
+    - `sdk.cases.get_case()`
+    - `sdk.case.update()`
+
+- `sdk.cases.file_events` methods for managing file events associated with a given case:
+    - `sdk.cases.file_events.add_event`
```

Comment:
missing `()` at the end of the file event methods

Reply:
added.